app/src/common/reporter.js

```diff
@@ -1,22 +1,31 @@
+import os from 'os';
 import isDev from 'electron-is-dev';
 import unhandled from 'electron-unhandled';
 
-let ravenClient;
+let Raven;
 
 function init() {
   if (!isDev) {
-    const raven = require('raven');
+    Raven = require('raven');
```

Comment:
Hmm, should we use `raven-js` in the renderer, and `raven` in the main process? 🤔